commands/checkout.go

```diff
@@ -92,18 +92,18 @@ func transformCheckoutArgs(args *Args) error {
 	repo, err := github.LocalRepo()
 	utils.Check(err)
 
+	var remoteRepo string
+
 	_, err = repo.RemoteByName(user)
 	if err == nil {
-		args.Before("git", "remote", "set-branches", "--add", user, branch)
-		remoteURL := fmt.Sprintf("+refs/heads/%s:refs/remotes/%s/%s", branch, user, branch)
-		args.Before("git", "fetch", user, remoteURL)
+		remoteRepo = user
```

Comment:
I don't think this is accurate anymore. We don't want to fetch from a remote that points to the head repo of the PR. We always want to fetch from the origin remote that represents the "main" project.

Thus, the process should be:

1. Get the **base** repo of the PR
2. Find the remote that points to that repo (will usually resolve to "origin" or "upstream")
3. If the remote can't be found, abort the action.